API/CLI.md

```diff
@@ -37,7 +48,7 @@ If `--name` is not specified, CLI will look for an environment variable, e.g. `e
 
 ### Starting Projects
 * `t2 init` in the current directory, create a package.json and index.js with Hello World code.
```

Comment:
While we're making an edit to this file, it would be great to link this part to the Hello World tutorial docs:

```
`t2 init` in the current directory, create a package.json and index.js with [Hello World code](http://tessel.github.io/t2-start/blinky.html).
```